src/CommunityToolkit.Aspire.Hosting.McpInspector/McpInspectorResource.cs

```diff
@@ -67,7 +67,7 @@ internal void AddMcpServer(IResourceWithEndpoints mcpServer, bool isDefault, Mcp
 
         McpServerMetadata item = new(
             mcpServer.Name,
-            mcpServer.GetEndpoint("http") ?? throw new InvalidOperationException($"The MCP server {mcpServer.Name} must have a 'http' endpoint defined."),
+            mcpServer.GetEndpoint("https") ?? mcpServer.GetEndpoint("http") ?? throw new InvalidOperationException($"The MCP server {mcpServer.Name} must have an 'https' or 'http' endpoint defined."),
```

Comment:
This never returns null

Reply:
What happens if there's no registered HTTPS endpoint though?